cli/azd/cmd/root.go

```diff
@@ -320,6 +321,11 @@ func NewRootCmd(
 			},
 		}).
 		UseMiddleware("hooks", middleware.NewHooksMiddleware)
+	root.
+		Add("add", &actions.ActionDescriptorOptions{
+			Command:        add.NewAddCmd(),
+			ActionResolver: add.NewAddAction,
+		})
```

Comment:
Can you add `HelpOptions` and `GroupingOptions` ?

Like
```
HelpOptions: actions.ActionHelpOptions{
			Description: getCmdEnvHelpDescription,
		},
		GroupingOptions: actions.CommandGroupOptions{
			RootLevelHelp: actions.CmdGroupManage,
		},
```

See the examples from other commands.  It should improve the docs for the help and you can add a few examples on the bottom

Reply:
Tracked with #4538